stacks/knot-cloud-core/README.md

```diff
@@ -0,0 +1,37 @@
+# knot-cloud-core
+
+### Configure DNS
+
+This stack uses [Traefik](https://traefik.io) as a reverse proxy for the services and requires that you configure your DNS server to point, at least, the **things**, **users** and **bt** subdomains to the machine where the stack is being deployed, so that it can route the requests to the appropriated service. It is possible to configure it to route by path or port, but instructions for that won't be provided here for brevity.
+
+If you don't have a domain or can't configure the main DNS server, you can configure a test domain in your machine before proceeding. Either setup a local DNS server, e.g. [bind9](https://wiki.debian.org/Bind9), or alternatively update your hosts file to include the following addresses:
+
+```
+127.0.0.1   things
+127.0.0.1   users
+127.0.0.1   bt
+```
+
+On Windows, the hosts file is usually located under `c:\Windows\System32\Drivers\etc\hosts`. On Unix systems, it is commonly found at `/etc/hosts`. Regardless of you operating system, administrator or super user privileges will be required.
+
+### Deploy: stage 1
+
+Stage 1 contains the core services. The next sections provide the instructions to configure and deploy them. Whenever a configuration file is mentioned, it refers to a file found at `stacks/prod/env.d`, for production mode, or `<path>/stack/dev/env.d`, for development mode.
+
+#### Deploy
+
+Deploy the stage 1 services:
+
+```bash
+docker stack deploy -c stage-1.yml knot-cloud-core
+```
+
+#### Verify
+
+Wait until all the services are started. You can check it by running:
+
+```bash
+docker stack services knot-cloud-core
+```
+
+And verifying that every service has one replica.
```

Comment:
I can't agree with that because a lot of services will be added here and all of them have the same importance. The main difference is that the `knot-cloud` provides a `healthcheck` endpoint for the intermediate `bootstrapping` stage, the developer/devOps guy need to know this information to follow the next instructions.

Reply:
I see your point and it makes sense :confetti_ball: